web/src/site-admin/SiteAdminAddExternalServicesPage.tsx

```diff
@@ -28,48 +22,45 @@ export class SiteAdminAddExternalServicesPage extends React.Component<Props> {
     /**
      * Gets the external service kind and add-service kind from the URL paramsters
```

Comment:
```suggestion
     * Returns the id of the external service from the URL parameters.
```

This comment seems outdated.